validation/kubernetes/fluxcd-v2/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {}
+
+module "fluxcd_v2" {
+  source = "../../../modules/kubernetes/fluxcd-v2"
+
+  azure_devops_pat       = "123456"
+  azure_devops_org       = "foo"
+  azure_devops_proj      = "bar"
+  github_org             = "foo"
+  github_app_id          = 123
+  github_installation_id = 123
+  github_private_key     = "foo" #tfsec:ignore:general-secrets-no-plaintext-exposure
+  environment            = "dev"
+  cluster_id             = "foobar"
+  namespaces             = []
```

Comment:
I'm not sure exactly how relevant it is in this context, but generally speaking testing and example values should be unique so as to minimize the risk of letting letting in the pasta cat. For example, a copy-paste error might produce a mixup between `azure_devops_org` and `github_org` both of which is "foo" here.